src/renderer/pages/Staking/Overview/Overview.tsx

```diff
@@ -237,6 +238,17 @@ export const Overview = () => {
         chain: activeChain,
         shards: uniqBy(shards, 'accountId'),
       });
+    } else if (path === Paths.RESTAKE) {
```

Comment:
Could be combined with other refactored operations, as I've mentioned in previous PRs